src/main/kotlin/it/pagopa/ecommerce/helpdesk/dataproviders/mongo/EcommerceTransactionDataProvider.kt

```diff
@@ -53,6 +56,8 @@ class EcommerceTransactionDataProvider(
         skip: Int,
         limit: Int
     ): Mono<List<TransactionResultDto>> {
+        val confidentialMailUtils =
```

Comment:
add a brief comment explaining why this bean is instantiated for each request and not used as singleton through CDI

Reply:
Fixed with 5fc1af38fab1f5f3ce183fbdb5b54a0a742d2a27